From 283a1deb6304b5452ca26270f268f3d07fa9ddce Mon Sep 17 00:00:00 2001
From: Bin Han <bin.b.han@intel.com>
Date: Sat, 10 Oct 2015 11:20:48 +0800
Subject: [PATCH] imx: disable VCM power operation by default

Port from R6_legacy for BTNS.
Unless SW explicitly specifies the VCM module, do not do
VCM power operations.

Change-Id: Icfed340d2be16d6920ef84ceafdcc45dcdd774fe
Original-Tracked-On: #H1504074607
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-7449
Signed-off-by: Bin Han <bin.b.han@intel.com>
Reviewed-on: https://android.intel.com:443/422127
Signed-off-by: Zhang, Guoqing <guoqingx.zhang@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/80828
Build: Dessert M BXT RVP (SC)
Build: Dessert M CHT FFD (SC)
Build: Dessert M COHO (SC)
Integrator: WeiX Meng <weix.meng@intel.com>
Validation-Android: Wei Tao <wei.tao@intel.com>
Approver: Tian Shu Qiu <tian.shu.qiu@intel.com>
Reviewed-by: WeiX Meng <weix.meng@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/imx/imx.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/imx/imx.h b/drivers/external_drivers/camera/drivers/media/i2c/imx/imx.h
index 1e2c0c3..6f8f211 100644
--- a/drivers/external_drivers/camera/drivers/media/i2c/imx/imx.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/imx/imx.h
@@ -694,8 +694,8 @@ struct imx_vcm imx_vcms[] = {
 		.t_vcm_timing = dw9718_t_vcm_timing,
 	},
 	[IMX_ID_DEFAULT] = {
-		.power_up = vcm_power_up,
-		.power_down = vcm_power_down,
+		.power_up = NULL,
+		.power_down = NULL,
 		.t_focus_abs_init = NULL,
 	},
 };
-- 
1.9.1

